setup.cfg

```diff
@@ -29,6 +29,7 @@ package_dir =
     =src
 # DON'T CHANGE THE FOLLOWING LINE! IT WILL BE UPDATED BY PYSCAFFOLD!
 setup_requires = pyscaffold>=3.2a0,<3.3a0
+                 numpy>=1.7.0
```

Comment:
will pyscaffold update this line in the future and cause the package to break?

Reply:
in pyscaffold 4 this won't work anymore, and this should all be put in pyproject.toml, which we don't have yet. I would suggest to leave it but open a new issue (this line is also more of a fix to #208 and should actually be in a separate PR).